### PR TITLE
adding sraid checks to datadog for DBMs

### PR DESCRIPTION
**Pull request by joanocallaghan**
@udemy/sre 

Adding software raid monitor from https://github.com/toksvaeth/datadog-plugin-md/blob/master/checks.d/md.py to monitor the flash drives on the DBM hosts. 

Note - I've already installed this manually on the DBMs. 

From trello - 
https://trello.com/c/44yJWJRO/821-add-monitor-for-software-raid-to-dbm-hosts-for-flash-drives

### CODE REVIEW

**Comment by joanocallaghan** [comment]
@tmcevoy14 do i even need to put in the .yaml file if it's being generated by the ansible group_vars part?

File: files/md.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+init_config:
```
Comment (tmcevoy14) [inline files/md.yaml]: You don't need this file here since you're defining it in group_vars/mysql-va/vars :)

**Comment by tmcevoy14** [comment]
@joanocallaghan you'll have to hack on Check Agent task to handle the dif paths on the mysql hosts *le sigh*

Something like this maybe? 
 
- name: Create a check agent for each Datadog check
  copy:
    src={{ item }}.py
    dest=/etc/dd-agent/checks.d/{{ item }}.py
    owner={{ datadog_user }}
    group={{ datadog_group }}
  with_items: '{{ datadog_check_agents }}'
  notify:
   - restart datadog-agent
  when: mysql_server_monitoring is not defined

- name: Create a check agent for each Datadog check on MySql hosts
  copy:
    src={{ item }}.py
    dest=/opt/datadog-agent/agent/checks.d{{ item }}.py
    owner={{ datadog_user }}
    group={{ datadog_group }}
  with_items: '{{ datadog_check_agents }}'
  notify:
   - restart datadog-agent
  when: mysql_server_monitoring is defined and mysql_server_monitoring

**Comment by joanocallaghan** [comment]
Not making the changes regarding file location mentioned above by Tracey as the live servers already have this check in place, put there manually, and if we run this playbook against any new mysql hosts that have been ansibilized, they will have the correct datadog version and the file location will be correct.

**Pull request merged**